articles/aws-api-setup.md

```diff
@@ -19,7 +19,7 @@ This is a one time setup
   ![](/media/articles/aws-api-setup/aws-api-setup-2.png)
 
 <% if (account.userName) { %>
-3. Download the metadata from <a href="https://@@account.namespace@@/samlp/metadata/@@account.clientId@@" target="_new">https://@@account.namespace@@/samlp/metadata/@@account.clientId@@</a> and upload it here.
+3. Download the metadata from <a href="https://@@account.namespace@@/samlp/metadata/@@account.clientId@@">https://@@account.namespace@@/samlp/metadata/@@account.clientId@@</a> and upload it here.
```

Comment:
Hmm. I prefer the other way :).

I'd like all links to ALWAYS open in another tab. What we want is for users to always continue reading Auth0 and then follow up on the other links and come back to Auth0. If we go to the other link in the same page, we might be loosing a lot of people.

What do you think?